doc/manuals/admin/database_model.md

```diff
@@ -322,29 +322,29 @@ Fields:
 Example document:
 
 ```
- {
```

Comment:
JSON hasn't changed actually, but for some reason github shows this as change...

NTC